src/store/modules/ability.js

```diff
@@ -59,7 +59,7 @@ export const useAbilityStore = defineStore(
 					})
 			},
 
-			saveAbility() {
+			saveAbility(abilityItem) {
 				if (!this.abilityItem) {
```

Comment:
Je moet deze if statement nog veranderen om te checken of `abilityItem` falsy is, niet `this.abilityItem`